app.py

```diff
@@ -5,7 +5,7 @@
 
 @app.route("/")
 def hello_world():
-    return "<p>Hello, World!</p>"
+    return "<p>Hello, Denis!</p>"
```

Comment:
Hi @MuhanovAndrey, can you please greet yourself?

Reply:
one second